lib/protobuf/active_record/transformer.rb

```diff
@@ -0,0 +1,22 @@
+module Protobuf
+  module ActiveRecord
+    class Transformer
+      attr_accessor :callable, :options
+
+      def initialize(callable, options = {})
+        @callable = callable
+        @options = options
+      end
+
+      def call(proto)
+        callable.call(proto)
+      end
+
+      def nullify?(proto)
+        return false unless options[:nullify_on]
+
+        proto.field?(:nullify) && proto.nullify.include?(options[:nullify_on].to_s)
```

Comment:
Do you think it would be beneficial to warn here if the `nullify_on` option is specified, but the message doesn't respond to `nullify`? It might help people figure out why fields aren't being set to nil more quickly.